firestore/google/cloud/firestore_v1beta1/field_path.py

```diff
@@ -0,0 +1,386 @@
+# Copyright 2017 Google LLC All rights reserved.
```

Comment:
Current header format is `# Copyright 2018 Google LLC`. Did you pick 2017 as that was the author date of this code originally?

Reply:
I just copied the header from `_helpers.py`.  ISTM that the original copyright assertion applies, although I don't really care one way or the other.

Reply:
091abe9